subprojects/problems/src/main/java/org/gradle/api/problems/ProblemBuilderDefiningDocumentation.java

```diff
@@ -19,13 +19,24 @@
 import org.gradle.api.Incubating;
 
 /**
- * Builder that is part of the {@link ProblemBuilder} concept.
+ * Builder for {@link Problem} instances requiring the specification of related documentation.
```

Comment:
🤔 IMO this is a bit hard to read, my bid is something like this:
```suggestion
 * {@link Problem} instance builder requiring the specification of documentation.
```

Reply:
This OFC applies to all the builder javadocs

Reply:
👍